pkg/registry/core/pod/rest/log.go

```diff
@@ -86,15 +94,34 @@ func (r *LogREST) Get(ctx context.Context, name string, opts runtime.Object) (ru
 		return nil, err
 	}
 	return &genericrest.LocationStreamer{
-		Location:        location,
-		Transport:       transport,
-		ContentType:     "text/plain",
-		Flush:           logOpts.Follow,
-		ResponseChecker: genericrest.NewGenericHttpResponseChecker(api.Resource("pods/log"), name),
-		RedirectChecker: genericrest.PreventRedirects,
+		Location:                    location,
+		Transport:                   transport,
+		ContentType:                 "text/plain",
+		Flush:                       logOpts.Follow,
+		ResponseChecker:             genericrest.NewGenericHttpResponseChecker(api.Resource("pods/log"), name),
+		RedirectChecker:             genericrest.PreventRedirects,
+		TLSVerificationErrorCounter: podLogsTLSFailure,
 	}, nil
 }
 
+func countSkipTLSMetric(insecureSkipTLSVerifyBackend bool) {
+	usageType := usageEnforce
+	if insecureSkipTLSVerifyBackend {
+		if utilfeature.DefaultFeatureGate.Enabled(features.AllowInsecureBackendProxy) {
+			usageType = usageSkipAllowed
+		} else {
+			usageType = usageSkipDenied
+		}
+	}
+
+	counter, err := podLogsUsage.GetMetricWithLabelValues(usageType)
+	if err != nil {
+		utilruntime.HandleError(err)
+		return
+	}
+	counter.Inc()
+}
```

Comment:
question: I assume this is equivalent to `podLogsUsage.WithLabelValues(usageType).Inc()` or are there some subtleties?

Reply:
> question: I assume this is equivalent to `podLogsUsage.WithLabelValues(usageType).Inc()` or are there some subtleties?

podLogsTLSFailure.Inc().  the bare counter doesn't have labels.